docs/user_guide/using_the_library.rst

```diff
@@ -32,6 +32,16 @@ The most basic function to use when inspecting a single NWBFile is the
 
 This returns a ``list`` of :py:class:`~nwbinspector.register_checks.InspectorMessage` objects.
 
+If you have an ``NWBFile`` object in memory, you can run:
+
+.. code-block:: python
+
+    from nwbinspector import available_checks, run_checks
+    from pynwb import NWBHDF5IO
+
+    with NWBHDF5IO(...) as io:
```

Comment:
```suggestion
    with NWBHDF5IO(path="path_to_single_nwbfile", mode="r", load_namespaces=True) as io:
```

The rest of the docs pretend to have a file path specified. Should also include the `load_namespaces` stuff so people can just copy/paste directly from here and have it always work